### ISSUE TITLE
Smart ctrl+c | copy text or kill process

### ISSUE THREAD

**Issue by j-urobles** [warpdotdev/Warp#4539]
### Describe the solution you'd like?

If you have some text selected CTRL+C will copy if not it will kill running process

### How important is this feature to you?

1 (Not too important)

### Warp Internal (ignore) - linear-label:39cc6478-1249-4ee7-950b-c428edfeecd1

None

**Comment by dannyneira** [comment]
Thanks for this feature request! 

To anyone else interested in this feature, please add a 👍 to the original post at the top to signal that you want this feature, and subscribe if you'd like to be notified.

**Comment by sjmikler** [comment]
This feature is common in other terminals, like Windows Terminal or Terminator. In Terminator it is called "Smart copy".

Please bring it to Warp.

**Comment by stone-w4tch3r** [comment]
Yep. This is built-in feature of powershell, and makes live in windows much easier than in linux. Nowadays I often open the developer menu by accidentally pressing ctrl+shift+c in the browser. For me this feature will have 3/4 importance rating.
